src/task 25/task25.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<html>
+    <head>
+        <meta charset="utf-8">
+    </head>
+    <body>
+        <input type="text">
+        <script>
+            var field = document.querySelector("input");
+            //charCodeAt(Q) = 81; charCodeAt(W) = 87; charCodeAt(X) = 88;
+            field.addEventListener("keydown", function(event) {
+                if (event.keyCode == 81 || event.keyCode == 87 || event.keyCode == 88){
+                    event.preventDefault();
+                };
+            });
```

Comment:
Please make your solution is more generic and flexible - it should become reusable function, receiving all the hard-coded values as configuration object. For example, here, you can pass 
- node id to extend functionality for
- array of characters to censor.